src/store/index.ts

```diff
@@ -0,0 +1,36 @@
+import { connectReducer, getConnectedStore } from '@onaio/connected-reducer-registry';
```

Comment:
Let us create an issue to move this into a package.

Reply:
link to issue: https://github.com/OpenSRP/web/issues/23